### ISSUE TITLE
[Request]: Drag-and-Drop to reorder pictures

### ISSUE THREAD

**Issue by theoteno** [pachli/pachli-android#1187]
In a web browser on Mastodon, you can reorder pictures by dragging pictures before/after an another image. It's a Quality of Life improvement like that I'd like to see in Pachli, as the Pixelfed app also have this feature to my knowledge!

Also would help, till [this Mastodon bug](https://github.com/mastodon/mastodon/issues/24462) is not fixed...

**Comment by theoteno** [comment]
The Pixelfed app might not have reordering for multiple images uploaded, but about to confirm that soon.

**Comment by theoteno** [comment]
While the Pixelfed app didn't even support uploading multiple images at once, on the Pixelfed website reordering works with arrows below the images. That might be also a good solution.

**Comment by nikclayton** [comment]
You mean to reorder images as part of writing a new post?

**Comment by theoteno** [comment]
Yes! I guess it could be done with other media files, but I usually share 4 screenshots (4 images is the current limit on Mastodon per toot) in one post when I played some games. Only 1 animated GIF, video and audio can be attached to a post on Mastodon currently, but it seems like those could be uploaded together.
